### PR TITLE
Bluetooth: controller: Enable GPIO debug pins for nrf52833 DK

### PR DESCRIPTION
**Pull request by ppryga-nordic**
Enable possiblity to use debug pins for controller debugging
purposes with nrf52833dk_nrf52833 board.

Signed-off-by: Piotr Pryga <piotr.pryga@nordicsemi.no>

### CODE REVIEW

File: subsys/bluetooth/controller/Kconfig.ll_sw_split [hunk 1]
```diff
@@ -664,7 +664,7 @@ config BT_CTLR_PROFILE_ISR
 
 config BT_CTLR_DEBUG_PINS
 	bool "Bluetooth Controller Debug Pins"
-	depends on BOARD_NRF51DK_NRF51422 || BOARD_NRF52DK_NRF52832 || BOARD_NRF52DK_NRF52810 || BOARD_NRF52840DK_NRF52840 || BOARD_NRF5340DK_NRF5340_CPUNET || BOARD_RV32M1_VEGA
+	depends on BOARD_NRF51DK_NRF51422 || BOARD_NRF52DK_NRF52832 || BOARD_NRF52DK_NRF52810 || BOARD_NRF52840DK_NRF52840 || BOARD_NRF5340DK_NRF5340_CPUNET || BOARD_RV32M1_VEGA || BOARD_NRF52833DK_NRF52833
```
Comment (cvinayak) [inline subsys/bluetooth/controller/Kconfig.ll_sw_split:667]:
```suggestion
	depends on BOARD_NRF51DK_NRF51422 || BOARD_NRF52DK_NRF52832 || BOARD_NRF52DK_NRF52810 || BOARD_NRF52840DK_NRF52840 || BOARD_NRF52833DK_NRF52833 || BOARD_NRF5340DK_NRF5340_CPUNET || BOARD_RV32M1_VEGA
```

**Review by cvinayak — CHANGES_REQUESTED** [review]
Minor change requested

File: subsys/bluetooth/controller/Kconfig.ll_sw_split (hunk 1 above)
Comment (ppryga) [reply subsys/bluetooth/controller/Kconfig.ll_sw_split]: moved BOARD_NRF52833DK_NRF52833 up in the depends on list

**Comment by ppryga** [comment]
@cvinayak requested change is ready.

**Review by cvinayak — APPROVED** [review]

**Review by carlescufi — APPROVED** [review]

**Pull request merged**